### PR TITLE
Replace ASA mutations with Formations API in e2e tests

### PR DESCRIPTION
**Pull request by ivantenevvasilev**
Changes proposed in this pull request:
- replace createAutomaticScenarioAssignment with assignFormation in e2e tests
- replace deleteAutomaticScenarioAssignmentForScenario with unassignFormation in e2e tests
- replace deleteAutomaticScenarioAssignmentsForSelector with unassignFormation in e2e tests

**Pull Request status**
- [x] Implementation
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

File: tests/director/tests/formation_api_test.go [hunk 1]
```diff
@@ -144,10 +144,10 @@ func TestRuntimeFormationFlow(t *testing.T) {
 		assert.Equal(t, asaFormation, deleteASAFormation.Name)
 	}()
 
-	asaInput := fixtures.FixAutomaticScenarioAssigmentInput(asaFormation, selectorKey, subaccountID)
+	formationInput := graphql.FormationInput{Name: asaFormation}
 	t.Log("Creating ASA")
-	fixtures.CreateAutomaticScenarioAssignmentInTenant(t, ctx, certSecuredGraphQLClient, asaInput, tenantId)
-	defer fixtures.DeleteAutomaticScenarioAssigmentForSelector(t, ctx, certSecuredGraphQLClient, tenantId, *asaInput.Selector)
+	fixtures.AssignFormationWithTenantObjectType(t, ctx, certSecuredGraphQLClient, formationInput, subaccountID, tenantId)
+	defer fixtures.UnassignFormationWithTenantObjectType(t, ctx, certSecuredGraphQLClient, formationInput, subaccountID, tenantId)
```
Comment (DimitarPetrov) [inline tests/director/tests/formation_api_test.go:150]: You can use clean up to make it more robust

File: tests/ord-service/tests/api_test.go [hunk 2]
```diff
@@ -267,9 +267,9 @@ func TestORDService(t *testing.T) {
 	defer fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, certSecuredGraphQLClient, tenantFilteringTenant, []string{"DEFAULT"})
 
 	// create automatic scenario assigment for subTenant
-	asaInput := fixtures.FixAutomaticScenarioAssigmentInput(scenarioName, selectorKey, subTenantID)
-	fixtures.CreateAutomaticScenarioAssignmentInTenant(t, ctx, certSecuredGraphQLClient, asaInput, tenantFilteringTenant)
-	defer fixtures.DeleteAutomaticScenarioAssignmentForScenarioWithinTenant(t, ctx, certSecuredGraphQLClient, tenantFilteringTenant, scenarioName)
+	formationInput := directorSchema.FormationInput{Name: scenarioName}
+	fixtures.AssignFormationWithTenantObjectType(t, ctx, certSecuredGraphQLClient, formationInput, subTenantID, tenantFilteringTenant)
+	defer fixtures.UnassignFormationWithTenantObjectType(t, ctx, certSecuredGraphQLClient, formationInput, subTenantID, tenantFilteringTenant)
```
Comment (DimitarPetrov) [inline tests/ord-service/tests/api_test.go:272]: Cleanup

File: tests/pkg/fixtures/scenario_assignment_requests.go [hunk 3]
```diff
@@ -11,16 +11,6 @@ import (
...
-	return graphql.AutomaticScenarioAssignmentSetInput{
-		ScenarioName: automaticScenario,
-		Selector: &graphql.LabelSelectorInput{
-			Key:   selectorKey,
-			Value: selectorValue,
-		},
-	}
-}
-
 func FixCreateAutomaticScenarioAssignmentRequest(automaticScenarioAssignmentInput string) *gcli.Request {
```
Comment (DimitarPetrov) [inline tests/pkg/fixtures/scenario_assignment_requests.go:14]: This can be deleted as well.

File: tests/tenant-fetcher-job/tests/handler_test.go [hunk 4]
```diff
@@ -286,9 +286,9 @@ func TestMoveSubaccountsFailIfSubaccountHasFormationInTheSourceGA(t *testing.T)
 	fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, certSecuredGraphQLClient, defaultTenantID, []string{"DEFAULT", scenarioName})
 	defer fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, certSecuredGraphQLClient, defaultTenantID, []string{"DEFAULT"})
 
-	asaInput := fixtures.FixAutomaticScenarioAssigmentInput(scenarioName, "global_subaccount_id", subaccountExternalTenants[0])
-	fixtures.CreateAutomaticScenarioAssignmentInTenant(t, ctx, certSecuredGraphQLClient, asaInput, defaultTenantID)
-	defer fixtures.DeleteAutomaticScenarioAssignmentForScenarioWithinTenant(t, ctx, certSecuredGraphQLClient, defaultTenantID, scenarioName)
+	formationInput := graphql.FormationInput{Name: scenarioName}
+	fixtures.AssignFormationWithTenantObjectType(t, ctx, certSecuredGraphQLClient, formationInput, subaccountExternalTenants[0], defaultTenantID)
+	defer fixtures.UnassignFormationWithTenantObjectType(t, ctx, certSecuredGraphQLClient, formationInput, subaccountExternalTenants[0], defaultTenantID)
```
Comment (DimitarPetrov) [inline tests/tenant-fetcher-job/tests/handler_test.go:291]: Cleanup

**Review by DimitarPetrov — CHANGES_REQUESTED** [review]
TestRuntimeUnregisterDeletesScenarioAssignments still uses createAutomaticScenarioAssignment

**Review by DimitarPetrov — APPROVED** [review]

**Review by desislavaa — APPROVED** [review]

**Pull request merged**